File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums = sorted(nums)
        if len(nums) == 1:
            return 1
        ans = 0
        curr = 1
        for i in range(1, len(nums)):
            if nums[i] - nums[i-1] == 1:
                curr += 1
                if curr > ans:
                    ans = curr
            elif nums[i] == nums[i-1]:
                if curr > ans:
                    ans = curr
            else:
                curr = 1
        
        return ans
            

File: Data Structures & Algorithms/concatenation-of-array/submission-4.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] ans = new int[2*n];
        for(int i = 0; i < n; i++){
            int curr = nums[i];
            ans[i] = curr;
            ans[i + n] = curr;
        }
        return ans;
    }
}